packages/rpc-core/src/rpc-methods/simulateTransaction.ts

```diff
@@ -0,0 +1,160 @@
+import { Base58EncodedAddress } from '@solana/addresses';
+import { Base64EncodedWireTransaction } from '@solana/transactions';
+
+import { TransactionError } from '../transaction-error';
+import {
+    AccountInfoBase,
+    AccountInfoWithBase64EncodedData,
+    AccountInfoWithBase64EncodedZStdCompressedData,
+    AccountInfoWithJsonData,
+    Base58EncodedBytes,
+    Base64EncodedDataResponse,
+    Commitment,
+    Slot,
+    U64UnsafeBeyond2Pow53Minus1,
+} from './common';
+
+type SimulateTransactionConfigBase = Readonly<{
+    /**
+     * Commitment level to simulate the transaction at
+     * @default finalized
```

Comment:
Ahh I wonder if we should start tagging all of these configs haha.

btw, looks like it's actually [`defaultValue`](https://typedoc.org/tags/defaultValue/)

Reply:
TY! Probably not worth adding it to all of them, just improve docstrings as we're working on stuff over time. But I should probably fix the 4 other places I've used `@default`!